wiki/en/Software-Manual.md

```diff
@@ -101,23 +101,25 @@ If you have set your Audio Channel to Stereo or Stereo Out in your Settings, you
 
 If you see a "mute" icon above a user, it means that person cannot hear you. Either they have muted you, soloed one or more users not including you, or have set your fader in their mix to zero.
 
+Users are listed left to right in the order that they connect. You can change the sort order using the Edit option in the application menu.
```

Comment:
I don't think that's necessarily true all the times (just most of the times) since old server slots might get filled by newcomers. Although it's mostly ok, I'd rather not remove that.

Reply:
I could add "Users are _usually_..." if you think that's accurate. I haven't removed anything in this PR, just moved this up. What do you think I've removed?

Reply:
Ah sorry. I edited the comment in a wrong way. It should say that I'd rather remove this paragraph.

Reply:
It does reveal the re-ordering feature. But it says it's on the Edit menu, when in fact it appears to be on the View menu. 

"Users are usually listed left-to-right in the order that they connect. You can sort by name, instrument, group, or city using the View menu."

This kind of page is typically rather comprehensive.